🚨Do not merge until Oct 18 launch - IDE GA docs (#2152)
fishtown-analytics/docs.getdbt.com: website/docs/docs/get-started/dbt-cloud-tips.md

# dbt Cloud tips

> The Cloud IDE is fully equipped with keyboard shortcuts, features, and development tips to help you to work like a dbt Cloud power user!  Use this Cloud IDE cheat sheet to help you save time and quickly reference.

Reviewer: ```suggestion
The Cloud IDE provides keyboard shortcuts, features, and development tips to help you work faster and be more productive. Use this Cloud IDE cheat sheet to help you quickly reference some common operations.
```

## Most popular

**Organize tabs**

> You can move your tabs around to reorganize your work in the IDE. You can also right-click on a tab to close multiple tabs or view and select a list of actions to take. Right click in the editor to access other options, including text wrap.

Reviewer: ```suggestion
You can move your tabs around to reorganize your work in the IDE. You can also right-click on a tab to close multiple tabs or view and select a list of actions to take. Right-click in the editor to access other options, including text wrap.
```

**Drag and drop**

> You can also drag and drop files in the file tree or in the editor. Use the file breadcrumb on the top of the IDE for quick, linear navigation. You can access adjacent files in the same file by right-clicking on the breadcrumb file.

Reviewer: ```suggestion
You can also drag and drop files in the file tree or in the editor. Use the file breadcrumb on the top of the IDE for quicker navigation. You can access adjacent files in the same file by right-clicking on the breadcrumb file.
```

**Search across files**

You can quickly search over all files in the IDE on your current project. To search, open the search bar by pressing Command-O to find text across all files in your current project and write your filename. You can view the results under the search text, which are grouped into files containing the match. You can click on the results to view them in the IDE.

**Command bar and status**

> You can run commands from the command bar at the bottom of the IDE or using the **Build** button. Use the [rich model selection syntax](/docs/reference/node-selection/syntax) to [run dbt commands](/docs/reference/dbt-commands) directly within dbt Cloud. You can also view the history, status, and logs of previous runs by clicking **Runs**.

Reviewer: ```suggestion
You can run commands from the command bar at the bottom of the IDE or by using the **Build** button. Use the [rich model selection syntax](/docs/reference/node-selection/syntax) to run [dbt commands](/docs/reference/dbt-commands) directly within dbt Cloud. You can also view the history, status, and logs of previous runs by clicking **Runs**.
```

## IDE Keyboard shortcuts

There are default keyboard shortcuts that can help make development more productive and easier for everyone.

- Command-O or Control-O to select a file to open
- Command-P or Control-P to see command palette

> - Hold Option-click-on-an-area to select multiple lines and perform a multi-edit. Also, press Command-E to perform this on the terminal of your choice.

Reviewer: ```suggestion
- Hold Option-click-on-an-area to select multiple lines and perform a multi-edit. You can also press Command-E to perform this operation on the command line.
```

- Press Fn-F1 to view a list of the other editor shortcuts

> - Command-Enter or Control-Enter to Preview

Reviewer: what kind of preview? a preview of what?

- Command-Shift-Enter or Control-Shift-Enter to Compile
- Highlight a portion of code and use the above shortcuts to Preview or Compile code

> - Type double underscore (__) in the IDE to reveal a list of dbt functions

Reviewer: ```suggestion
- Enter two underscores (__) in the IDE to reveal a list of dbt functions
```

## Multiple selections

> You can make multiple selections for quick and simultaneous edits. The below commands are a common way to add more cursors and allow you to insert cursors below or above with ease.

Reviewer: ```suggestion
You can make multiple selections for small and simultaneous edits. The below commands are a common way to add more cursors and allow you to insert cursors below or above with ease.
```

Reviewer: or "small and simultaneous edits"
